tests/acceptance/ignore-logs/new-agent.err

```diff
@@ -0,0 +1,13 @@
+| versions          - semgrep x.y.z on Python x.y.z
+| environment       - running in environment git, triggering event is 'unknown'
+| manage            - not logged in
+=== setting up agent configuration
+| using semgrep rules from https://semgrep.dev/c/p/r2c-ci
+| reporting findings introduced by these commits:
+|   * aaaaaaa add finding
+| looking at 2 changed paths
+=== looking for current issues in 2 files
```

Comment:
Why is this 2 files? I just thought it was always 1 file now

Reply:
Or is that only for full scans

Reply:
I'm pretty sure that's only for full scans, where we just pass the whole repo to the CLI. Diff scans have to pass specific individual files.